packages/edit-site/src/components/page-patterns/use-patterns.js

```diff
@@ -25,8 +26,9 @@ const EMPTY_PATTERN_LIST = [];
 
 const selectTemplateParts = createSelector(
 	( select, categoryId, search = '' ) => {
-		const { getEntityRecords, isResolving: isResolvingSelector } =
-			select( coreStore );
+		const { getEntityRecords, isResolving: isResolvingSelector } = unlock(
+			select( coreStore )
+		);
```

Comment:
I think there's no need to unlock this store. All requested selectors are public.